docs/ui/components/Collapsible/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { css } from '@emotion/react';
+import { borderRadius, iconSize, shadows, spacing, theme } from '@expo/styleguide';
+import React, { PropsWithChildren, ReactNode } from 'react';
+
+import { HEADLINE } from '~/ui/components/Text';
+import { durations } from '~/ui/foundations/durations';
+import { ChevronDownIcon } from '~/ui/foundations/icons';
+
+type CollapsibleProps = PropsWithChildren<{
+  /** The content of the collapsible summary */
+  summary: ReactNode;
+  /** If the collapsible should be rendered "open" by default */
+  open?: boolean;
+  testID?: string;
+}>;
+
+export function Collapsible({ summary, open, testID, children }: CollapsibleProps) {
+  return (
+    <details css={detailsStyle} open={open} data-testid={testID}>
+      <summary css={summaryStyle}>
+        <ChevronDownIcon css={markerStyle} size={iconSize.small} />
```

Comment:
Did want to prepare for a custom icon, instead of the default marker symbol. But we don't have the icon from the designs in here yet. Can change this to anything else, if preferred.

Reply:
We should have the icon in the designs as `<TriangleRightIcon />` and `<TriangleDownIcon />`. You can see all the icons and their names here: https://www.figma.com/file/wbPwIN1kV7wMgMtwHFvZrZ/Design-System?node-id=1166%3A345
![Icons](https://user-images.githubusercontent.com/6455018/152836195-be0ea7eb-d7e5-4d4f-ad17-8a66e582621a.png)